### ISSUE TITLE
forward_with_cond_scale(*) in classifier_free_guidance.py

### ISSUE THREAD

**Issue by xunguangwang** [lucidrains/denoising-diffusion-pytorch#155]
Hi, @lucidrains! In forward_with_cond_scale(*) function of classifier_free_guidance.py, it seems that the code `return null_logits + (logits - null_logits) * cond_scale` is not right. According to the original paper [(CLASSIFIER-FREE DIFFUSION GUIDANCE)](https://arxiv.org/abs/2207.12598) I think the code should be `logits + (logits - null_logits) * cond_scale`. Maybe I am wrong. Could you please respond to this question?

**Comment by lucidrains** [comment]
@xunguangwang it is equivalent, just my scale starts at 1, but theirs start at 0
